files/fr/web/html/link_types/dns-prefetch/index.html

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Types de liens : dns-prefetch'
+slug: Web/HTML/Link_types/dns-prefetch
+browser-compat: html.elements.link.rel.dns-prefetch
+translation_of: 'Web/HTML/Link_types/dns-prefetch'
+---
+<p><span class="seoSummary">Le mot-clé <strong><code>dns-prefetch</code></strong> de l'attribut {{HTMLAttrxRef("rel", "link")}} de l'élément {{HTMLElement("link")}} est une information destinée à indiquer aux navigateurs que la personne visitant le site va avoir besoin de charger des ressources provenant de l'origine de la ressource cible, et donc que le navigateur peut sans doute améliorer l'expérience de visite en effectuant la résolution DNS de cette origine de façon préventive.</span></p>
```

Comment:
`span`s with seoSummary, when they simply target the first paragraph can (and should) be removed.
They're an old way of explicitly setting the summary displayed on search engines. The thing is... this summary is automatically derived from the first paragraph.

Reply:
Ah! Good to know!
Removed in https://github.com/mdn/translated-content/pull/1794/commits/ddead30694dbe0471099bb3e5afc1b5d02f17cff